scene/resources/compressed_texture.h

```diff
@@ -40,6 +40,7 @@ class CompressedTexture2D : public Texture2D {
 
 public:
 	enum DataFormat {
+		DATA_FORMAT_UNDEFINED,
```

Comment:
This apparently broke compatibility for existing compressed textures, as it's reordering all enum values. New enum values should go to the end of the enum to avoid breaking compatibility.

(Haven't tested yet to confirm but I got multiple reports of a compat breakage on compressed textures introduced in today's merges.)

Reply:
I'll check it now

Reply:
@akien-mga, how can I push a fix?

Reply:
IIRC the best way to go about it (since this PR has been merged) is to make any fixes to a separate branch and PR them separately

Reply:
Okay, I'll make a new pull request now.

The problem is that the `UNDEFINED` image format is the format that all the old images are in. That's why I can't put it at the end of the list.

Reply:
I created a new pull-request: https://github.com/godotengine/godot/pull/86835

Reply:
I'm very sorry that this happened. I want to do everything to minimize the consequences. Perhaps even rollback this pull request if necessary